lib/spack/spack/build_systems/python.py

```diff
@@ -178,6 +181,35 @@ def install(self, spec, prefix):
         with working_dir(self.build_directory):
             pip(*args)
 
+    @property
+    def headers(self):
+        """Discover header files in platlib."""
+
+        root = inspect.getmodule(self).platlib
+        headers = find_all_headers(root)
```

Comment:
(question) there was some concern about whether this change would be disruptive; if I understand correctly, the CI generally resolved that, but would it be "extra safe" to fall back to `prefix.include` here (if it exists and is nonempty)?

Reply:
> In the case of py-greenlet, I almost wonder if it would be better to first check include and then check platlib if that's empty.

That would be fine with me, although I don't understand the advantage of doing that

My recommendation is to add `prefix.include` if it exists (in addition to `platlib` - unless e.g. in the case of `py-greenlet` that one symlinks to the other (in that case, we should either try to detect that, or stick with what you have now and just include one of platlib/prefix.include).